api/services/control/control.proto

```diff
@@ -66,13 +66,40 @@ message SolveRequest {
 }
 
 message CacheOptions {
+	// ExportRef is deprecated in favor or the new Exports since BuildKit v0.4.0.
+	// When ExportRef is set, the solver appends
+	// {.Type = "registry", .Attrs = ExportAttrs.add("ref", ExportRef)}
+	// to Exports for compatibility.
 	string ExportRef = 1;
```

Comment:
Should we rename these to `ExportRefDeprecated` etc.?

Reply:
renamed